src/tintx/objects.py

```diff
@@ -284,6 +312,7 @@ def write_tracks(
             "max": obj_props["field_max"],
             "mean": obj_props["field_mean"],
             "isolated": obj_props["isolated"][gi_obj],
+            "geometry": obj_props["geometry"],
```

Comment:
I think this should be a geo data frame. so instead `pd.DataFrame` using `gp.GeoDataFrame` where `gp` is goepandas. The reason for doing that is that geopandas can deal with projections (via the crs keyword). If we did that we'd have to include the Coordinate Reference System (crs) as a keyword to the user interface. For example, if users would like to track data from RCM output with rotated pole coordinate systems.